fulp_modules/main_features/bloodsuckers/code/bloodsucker/bloodsucker_life.dm

```diff
@@ -285,34 +304,43 @@
 
 /// Frenzy's End is in HandleStarving.
 /datum/antagonist/bloodsucker/proc/Frenzy_Start()
-	to_chat(owner.current, "<span class='userdanger'><FONT size = 3>Blood! You need Blood, now! You enter a total Frenzy!</span>")
-	to_chat(owner.current, "<span class='announce'>* Bloodsucker Tip: While in Frenzy, you instantly Aggresively grab, cannot speak, hear, get stunned, or use any powers outside of Feed.</span><br>")
-	/// Disable ALL Powers
-	for(var/datum/action/bloodsucker/power in powers)
-		if(power.active)
-			power.DeactivatePower()
-	owner.current.add_client_colour(/datum/client_colour/cursed_heart_blood)//bloodlust) <-- You can barely see shit, cant even see anyone to feed off of them.
-	ADD_TRAIT(owner.current, TRAIT_MUTE, BLOODSUCKER_TRAIT)
-	ADD_TRAIT(owner.current, TRAIT_DEAF, BLOODSUCKER_TRAIT)
-	ADD_TRAIT(owner.current, TRAIT_STUNIMMUNE, BLOODSUCKER_TRAIT)
-	/// Congratulations, you are the dumbest guy in Town.
-	if(HAS_TRAIT(owner.current, TRAIT_ADVANCEDTOOLUSER))
-		REMOVE_TRAIT(owner.current, TRAIT_ADVANCEDTOOLUSER, SPECIES_TRAIT)
+	if(my_clan == CLAN_BRUJAH)
+		to_chat(owner.current, "<span class='announce'>You enter a Frenzy!<br> \
+		* While in Frenzy, you gain the ability to instantly aggressively grab people, move faster and have no blood cost on abilities.<br> \
+		* In exchange, you will slowly gain Burn damage, be careful of how you handle it!<br> \
+		* To leave Frenzy, simply drink enough Blood (250) to exit.</span><br>")
+	else
+		to_chat(owner.current, "<span class='userdanger'><FONT size = 3>Blood! You need Blood, now! You enter a total Frenzy!</span>")
+		to_chat(owner.current, "<span class='announce'>* Bloodsucker Tip: While in Frenzy, you instantly Aggresively grab, cannot speak, hear, get stunned, or use any powers outside of Feed and Trespass (If you have it).</span><br>")
+		ADD_TRAIT(owner.current, TRAIT_STUNIMMUNE, BLOODSUCKER_TRAIT) // Brujah can control Frenzy properly, so they don't get any of the effects.
```

Comment:
isn't being immune to stuns the main benefit of being in a frenzy, though???

Reply:
You're feral when in Frenzy, Brujah can control their frenzy, so they arent uncontrollably feral and full of adrenaline.

Reply:
Forgot to mention that Brujah can use their powers while in a Frenzy, one of which is Fortitude, which itself gives bloodsucker_trait stun immunity